docs/install/install-from-packages.md

```diff
@@ -325,6 +346,10 @@ make install
 This will take a few minutes.
 ☕️
 First the backend, then the frontend will be installed.
+
+
+If mrs-developer is already installed, when the process completes successfully, it will exit with no message.
```

Comment:
I have not verified whether installing mrs-developer exits without a message. It used to exit with `✨  Done in 98.97s.`, but haven't tried it in a while. I'll go with what you say, "no message". Thus we can delete this line, as it is redundant to line 363 below.

Reply:
I took a look how and when mrs-developer is installed. It seems that nowadays mrs-developer is installed without a dialog if it is needed, which is the case if  a configuration file `mrs.developer.json` is present. 
So I'll kick out the lines about the mrs-developer dialog.

See Makefile: `npx -p mrs-developer`
```
.PHONY: develop
develop: ## Runs missdev in the local project (mrs.developer.json should be present)
	if [ -f $$(pwd)/tsconfig.json ]; then npx -p mrs-developer missdev --config=tsconfig.json --output=addons --fetch-https; else  npx -p mrs-developer missdev --config=jsconfig.json --output=addons --fetch-https;fi

```